### PR TITLE
Pytest Style: test_selections.py

### PR DESCRIPTION
**Pull request by utkbansal**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/utils/test_selections.py [hunk 1]
```diff
@@ -37,57 +38,55 @@
 from MDAnalysis.lib.util import NamedStream
 
 
-class _SelectionWriter(TestCase):
-    __test__ = False
+class _SelectionWriter(object):
+    # __test__ = False
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/utils/test_selections.py]: remove commented out line

File: testsuite/MDAnalysisTests/utils/test_selections.py [hunk 2]
```diff
@@ -189,7 +188,7 @@ def spt2array(line):
 
 
 class TestSelectionWriter_Jmol(_SelectionWriter):
-    __test__ = True
+    # __test__ = True
```
Comment (orbeckst) [inline testsuite/MDAnalysisTests/utils/test_selections.py]: remove comment (I assume that the test still gets executed?)

**Comment by jbarnoud** [comment]
Wouldn't this test suite benefit from parametrizing the base class instead of subclassing it?

File: testsuite/MDAnalysisTests/utils/test_selections.py [hunk 3]
```diff
@@ -100,7 +99,6 @@ def lines2one(lines):
 
 
 class TestSelectionWriter_Gromacs(_SelectionWriter):
```
Comment (kain88-de) [inline testsuite/MDAnalysisTests/utils/test_selections.py]: why are you using the composition here? We wanted to Avoid mixing classes.

**Comment by utkbansal** [comment]
@jbarnoud @kain88-de `_assert_selectionstring` method varies across classes. What do I do about that?

**Comment by utkbansal** [comment]
Isn't this a case similar to the coordinates tests?

**Comment by jbarnoud** [comment]
Hum... They indeed differ more than I initially thought, especially the gromacs one. @kain88-de ?

**Pull request merged**